packages/mongo-async/collection.js

```diff
@@ -745,16 +745,29 @@ Object.assign(Mongo.Collection.prototype, {
 
   // We'll actually design an index API later. For now, we just pass through to
   // Mongo's, but make it synchronous.
-  _ensureIndex(index, options) {
+    /**
+   * @summary Creates the specified index on the collection.
+   * @locus server
+   * @deprecated in 3.0
```

Comment:
like this @radekmie ?

Reply:
It's fine, but the `@method` is incorrect now (it says `createIndex` instead of `_ensureIndex`).